shared/folders/files/index.js

```diff
@@ -1,14 +1,90 @@
 // @flow
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
-import {Text} from '../../common-adapters'
+import {bindActionCreators} from 'redux'
+import {openInKBFS} from '../../actions/kbfs'
+import {navigateBack} from '../../actions/router'
+import Render from './render'
+import _ from 'lodash'
 
-// import Render from './render'
+import type {Folder} from '../list'
+
+type Props = $Shape<{
+  folder: ?Folder,
+  path: string,
+  username: string,
+  navigateBack: () => void,
+  openInKBFS: (path: string) => void
+}>
+
+type State = {
+  visiblePopupMenu: boolean
+};
+
+class Files extends Component<void, Props, State> {
+  state: State;
+
+  _checkFolderExistance (props) {
```

Comment:
Thanks. Two strikes with spelling in the past 24 hours, I'm going to be installing a spell checker ASAP.